nullaway/build.gradle

```diff
@@ -66,13 +66,29 @@ test {
   maxHeapSize = "1024m"
   if (!JavaVersion.current().java9Compatible) {
     jvmArgs "-Xbootclasspath/p:${configurations.errorproneJavac.asPath}"
+  } else {
+    // to expose necessary JDK types on JDK 16+; see https://errorprone.info/docs/installation#java-9-and-newer
```

Comment:
Makes sense. Wonder if we might need to change our own READMEs here once we officially support JDK 17.

Reply:
Yes, we need to make a pass on our READMEs, particularly for non-Android support; see #467 (which I really need to get back to).  For Gradle users, I think these flags all get added automatically by `gradle-errorprone-plugin`.  We need to add them manually here since our unit test runs don't get configured using that plugin